extension/fts/test/test_files/basic.test

```diff
@@ -0,0 +1,346 @@
+-DATASET CSV empty
+
+--
+
+-CASE Basic
+
+-STATEMENT LOAD EXTENSION '${KUZU_ROOT_DIRECTORY}/extension/fts/build/libfts.kuzu_extension';
+---- ok
+
+-STATEMENT CREATE NODE TABLE Book (ID SERIAL, abstract STRING, author STRING, title STRING, PRIMARY KEY (ID));
+---- ok
+-STATEMENT CREATE (b:Book {abstract: 'An exploration of quantum mechanics.', author: 'Alice Johnson', title: 'The Quantum World'});
```

Comment:
Can you replace those `CREATE STATEMENTS` with a dataset?
You can put all those data in a csv file and `CREATE TABLE` statements in the ddl file.
You can use `tinysnb` dataset as an example